sre_maturity.md

```diff
@@ -0,0 +1,115 @@
+# SRE Maturity Model
+
+## Phase 1: The ‘Crawl’ Phase
+
+### Overview
+
+You’re starting from zero, with a net new team.
+You may only have a small SRE team, or an SRE function within your development team.
+You and your team are technically capable of surviving.
+You are probably providing a ‘white glove’ experience to your users at this stage.
+
+### Characteristics
+
+#### People and Culture
+
+You have a strong SRE voice in your team, someone who advocates for reliability features.
+They establish people focused mantras like "It's OK to fail", "Hope is not a strategy" and "Trust by default & extend on it".
+Communication is important in your team and you hire diverse people based on soft skills & leadership rather than specific technologies.
+
+At this phase, having buy-in from executive level leadership is essential.
+Without this it will be very difficult to get their support later, and eventually their engagement.
+
+The product manager for you service should have a good understanding of SRE.
+They should help you advocate for service features that make the service more reliable.
+
+#### Observability
+
+The foundations for monitoring and observing your service are in place.
+Being able to measure something meaningful about your service is important, and supplements any feedback you get from initial users.
+A common way to do this is via metrics exposed by your service.
+
+You have a simple Service Level Objective (SLO) for you service to start out with.
+It doesn't have to be completely representative of what users expect from your service, but it forms the building blocks of an SLO review process and improving on them in time.
+Starting with a lower SLO than where you want to get to sets expectations early.
+
+#### “Admin” access to the service
```

Comment:
This is being called out because there are SRE teams somewhere who don't have this?

Reply:
@jeremyeder Good question.
This came up as a suggestion on 1 of the maturity calls as something for phase 1 that you absolutely need for your SRE team.

How about this?
Building on the comment from @billmvt below, the title of the section could change to something security focused. e.g. "Restricted access to the service"

Phase 1 proposes "production access restricted to a tightly scoped set of individuals".
Ad hoc security scans or pen tests would happen here too.

Phase 2 is 'shifting left' with scans in source code & 'shifting right' with regular or ongoing production scans. 
Couple this with 'security impact assessments' being considered at the design phase of features and architecture decisions.
Auditing of user access and having security compliance is a must in this phase.

Phase 3, I'm not sure but something  around "Automated Policy Enforcement".
I don't know what that means just yet.

Reply:
I've pushed some changes to the structure. There's a section for 'Security' phases of maturity now.